### DISCUSSION TITLE
Blue Screen of Death on any msys2 based install

### DISCUSSION THREAD

**Discussion by jasonbeverage** [microsoft/vcpkg#20135] [Q&A]
I'm getting a blue screen of death whenever I try to install anything that uses msys2.  I first noticed it when I was trying to install libiconv, but I'm also getting it when I try to install ffmpeg or anything msys2 based.  It starts to build and I see lots of cl.exe processes fire up, but then it eventually dies with  a KERNEL_MODE_HEAP_CORRUPTION blue screen and my machine restarts.

One of my coworkers had the same issue with ffmpeg on his machine awhile back but it seems to have corrected itself somehow and we don't really know what the actual was.

I've on Windows 10 and I have both VS 2017 and 2019 installed.

Any guidance would be appreciated!

**Comment by ThomasWilshaw** [comment]
I know this is a very old question but did you find a fix? I've been having the same issue recently.
